pkg/cmd/init.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright © 2019 Doppler <support@doppler.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/DopplerHQ/cli/pkg/configuration"
+	"github.com/DopplerHQ/cli/pkg/http"
+	"github.com/DopplerHQ/cli/pkg/printer"
+	"github.com/DopplerHQ/cli/pkg/utils"
+	"github.com/spf13/cobra"
+)
+
+// repoConfigFileName (doppler.yaml)
+const projectTemplateFileName = "doppler.yaml"
+
+var initCommand = &cobra.Command{
+	Use:   "init",
+	Short: "Automatically set up projects in your Doppler dashboard",
+	Args:  cobra.MaximumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		jsonFlag := utils.OutputJSON
+		defaultProjectTemplateFile := filepath.Join("./", projectTemplateFileName)
+		projectTemplateFile := utils.GetPathFlagIfChanged(cmd, "template", defaultProjectTemplateFile)
```

Comment:
Once we're specifying the default value:
```suggestion
		projectTemplateFile, err := utils.GetFilePath(cmd.Flag("template").Value.String())
```